lib/impress.constants.js

```diff
@@ -172,9 +173,8 @@ impress.DEFAULT_API = [
 
 // Preparing stack trace transformations
 impress.STACK_REGEXP = [
-  [api.path.dirname(__dirname) + api.common.pathSeparator + 'node_modules', ''],
-  [__dirname + api.common.pathSeparator, ''],
   [process.cwd() + api.common.pathSeparator + 'node_modules', ''],
```

Comment:
1. `process.cwd()` is not an equivalent change for `__dirname`.
2. On path concatenation:
  2.1. As for me, if we do some sort of cross-platform path concatenation, it's better to use `api.path.join` than to introduce a new constant and do it manually.
  2.2. This should be consistent across the codebase. Currently some code does `a + api.common.pathSeparator + b` and most of the code does `a + '/' + b`.